### ISSUE TITLE
Duplicate `id` in column name upon declaring a reference to `Identifier`

### ISSUE THREAD

**Issue by lundibundi** [metarhia/metasql#237]

**Comment by lundibundi** [comment]
@georgolden yep, wrong repo, moved to metasql now.
Though for metaschema there is still an issue with `d.ts` types which are not generated according to the cases here. Though it probably means that we have to move types generation here as well.

**Comment by georgolden** [comment]
Yeah, there was and there are still some issues with type generation, but  typings are not for metasql. It can be done, after we will meet agreement on current changes. We don't have an agreement about current implementation. I improved code quality, and we have no guidelines for static and private fields, and it's usage, so there can be even more refactoring within the current metaschema 2.0.0-alpha
